client/nginx.conf

```diff
@@ -12,6 +12,8 @@ http {
 
                 location / {
                         root /usr/src/app;
+                        add_header Cache-Control no-cache;
+                        expires 0;
```

Comment:
wouldn't this mean all files from `/` will not be cached? we just want `/index.html` to not be cached.